src/home/room_screen.rs

```diff
@@ -1231,47 +1234,56 @@ impl Widget for RoomScreen {
-                                    &tl_state.user_power,
-                                    item_drawn_status,
-                                    room_screen_widget_uid,
-                                )
+                            TimelineItemContent::MsgLike(msg_like_content) => {
+                               
+                                match &msg_like_content.kind {
+                                    MsgLikeKind::Message(message) => {
+                                        let prev_event = tl_idx.checked_sub(1).and_then(|i| tl_items.get(i));
+                                        populate_message_view(
+                                            cx,
+                                            list,
+                                            item_id,
+                                            room_id,
+                                            event_tl_item,
+                                            MessageOrSticker::Message(message, msg_like_content),
+                                            prev_event,
+                                            &mut tl_state.media_cache,
+                                            &tl_state.user_power,
+                                            item_drawn_status,
+                                            room_screen_widget_uid,
+                                        )
+                                    }
+                                    MsgLikeKind::Sticker(sticker) => {
+                                       let prev_event = tl_idx.checked_sub(1).and_then(|i| tl_items.get(i));
+                                        populate_message_view(
+                                            cx,
+                                            list,
+                                            item_id,
+                                            room_id,
+                                            event_tl_item,
+                                            MessageOrSticker::Sticker(sticker.content()),
+                                            prev_event,
+                                            &mut tl_state.media_cache,
+                                            &tl_state.user_power,
+                                            item_drawn_status,
+                                            room_screen_widget_uid,
+                                        ) 
+                                    }
+                                    MsgLikeKind::Redacted => populate_small_state_event(
+                                        cx,
+                                        list,
+                                        item_id,
+                                        room_id,
+                                        event_tl_item,
+                                        &RedactedMessageEventMarker,
+                                        item_drawn_status,
+                                    ),
+                                    _ => {
```

Comment:
Just a future todo for ourselves, which we should also file as an issue.
```suggestion
                                    // TODO: implement `Redacted`, `Poll`, and `UnableToDecrypt` types as regular Messages, not small state events.
                                    _ => {
```